LandSandBoat/server
# [core] [lua] [sql] Nuke nameflags, rework 0x037 and 0x00D (and anything else in the way of destruction)

**_I affirm:_**

## What does this pull request do?

Rework 2SC 0x037 based on https://github.com/atom0s/XiPackets/tree/main/world/server/0x0037
Rework 2SC 0x00D based on https://github.com/atom0s/XiPackets/tree/main/world/server/0x000D
Add sanity to codebase by removing devilish nameflags
Bunch of other stuff, like optimizing the size of 0x00D to generally use less bandwidth

## Steps to test these changes

Play the game and see nothing unusual --
Use all the old "nameflags" like d/c, mentor, /inv etc in search and see it work
